doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2578,7 +2578,7 @@ Bootloader
 
 The issues in this section are related to :ref:`app_bootloaders`.
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
```

Comment:
Issue has been fixed with PR https://github.com/nrfconnect/sdk-nrf/pull/12846.
The NCSDK-24203 is currently marked as Closed.